bin/seagull.js

```diff
@@ -96,43 +102,48 @@ function copyFilesToHtml(done) {
 }
 
 function readAllFiles(dir, callback) {
-  console.log('readAllFiles(): entry(%s, %s)', dir)
+  console.log('readAllFiles(): entry(%s)', dir)
```

Comment:
Removed a `%s` from this, since you were only passing one arg, and it was `console.log()`ging a literal `"%s"`.